tests/manage/mcg/test_host_node_failure.py

```diff
@@ -8,8 +8,7 @@
     ManageTest,
     skipif_ocs_version,
     skipif_vsphere_ipi,
-    tier4,
-    tier4a,
+    tier4b,
```

Comment:
I think we still want to keep the `tier4` marker

Reply:
IMO the whole purpose of segregating the tier4 testcases as node failures, pod failures and day2 operations is to achieve less run time . if we include tier4 we might not achieve the less run time and neatly segregated testcases. 
@keesturam at any point of time do we prefer running @tier4 testcases without specifying tier4a, tier4b or tier4c ? WDYT ?

Reply:
I believe the point Elad makes is that having tier4 marker would still allow us to run *all* tier4 tests without affecting tier4a, 4b, or 4c. The pipeline jbs would still use 4a, 4b and 4c markers.

Reply:
correct